tornado_http_auth.py

```diff
@@ -207,7 +207,7 @@ def authenticate_user(self, check_credentials_func, realm):
 
         auth_data = auth_header.split(None, 1)[-1]
         auth_data = base64.b64decode(auth_data).decode('ascii')
-        username, password = auth_data.split(':')
+        username, password = auth_data.split(':',maxsplit=1)
```

Comment:
`str.split()` on Python 21 doesn't accept keyword arguments. Just make it positional (i.e. `.split(':', 1) and I'll merge it. Also, add a space after the comma, please :)